bin/gpo

```diff
@@ -815,7 +816,7 @@ class gPodderCli(object):
         minarg, maxarg = len(args) - len(defaults), len(args)
 
         if len(command_line) < minarg or (len(command_line) > maxarg and
-                varargs is None):
+                                          varargs is None):
```

Comment:
can you split at the first or instead?

Reply:
I am not sure what you mean, 
had move this line so that is aligned with opening delimiter.

Reply:
Sorry, I mean split it like this at the 'or' instead of the 'and':
```
 if len(command_line) < minarg or
         (len(command_line) > maxarg and varargs is None):
```